packages/blocks/src/_specs/_specs.ts

```diff
@@ -61,6 +62,7 @@ const DocPageSpec: BlockSpec<PageRootBlockWidgetName> = {
       //   AFFINE_BLOCK_HUB_WIDGET
       // )}`,
       [AFFINE_MODAL_WIDGET]: literal`${unsafeStatic(AFFINE_MODAL_WIDGET)}`,
+      [AFFINE_PIE_MENU_WIDGET]: literal`${unsafeStatic(AFFINE_PIE_MENU_WIDGET)}`,
```

Comment:
This spec is used by affine and we need the designers and pms review this feature. So, maybe we can't add them here for now. Play we can enable them in our playground :)

Reply:
Hey, @doodlewind mentioned it. 
> Limiting the usage of this widget in edgeless mode should be expected.
I will fix it. Thanks!